pkg/connector/host_group.go

```diff
@@ -296,8 +438,9 @@ func newHostGroupGrantFromDN(hostGroupResource *v2.Resource, ipaUniqueID string,
 
 func hostGroupBuilder(client *ldap.Client, baseDN *ldap3.DN) *hostGroupResourceType {
 	return &hostGroupResourceType{
-		resourceType: resourceTypeHostGroup,
-		client:       client,
-		baseDN:       baseDN,
+		resourceType:   resourceTypeHostGroup,
+		client:         client,
+		baseDN:         baseDN,
+		ipaObjectCache: newIPAObjectCache(client, baseDN),
```

Comment:
Should this come as a dependency? so we can reuse from the previous cache since we use of 2 Builders

Reply:
updated